File: testcases/paper/FilterAttributes/FilterAttributes.go
package FilterAttributes

type ExternalClass struct {
    external int
}

var ExternalClassInstance ExternalClass

type FilterAttributes struct {
    i, j int
}

func (fa *FilterAttributes) A() int {
    fa.i = ExternalClassInstance.external
    return fa.i
}

func (fa *FilterAttributes) B() int {
    fa.j = ExternalClassInstance.external
    return fa.j
}

File: testcases/paper/FilterAttributes/FilterAttributes.py
class ExternalClass:
    external = None

class FilterAttributes:
    def __init__(self):
        self.i = None
        self.j = None

    def A(self):
        self.i = ExternalClass.external
        return self.i

    def B(self):
        self.j = ExternalClass.external
        return self.j
